lib/objects/objectsUtils.js

```diff
@@ -471,7 +473,9 @@ function sanitizePath(id, name, callback) {
         return;
     }
 
-    id = id.replace(/\.\./g, ''); // do not allow to write in parent directories
+    if (id) {
+        id = id.replace(/\.\./g, ''); // do not allow to write in parent directories
```

Comment:
I'm guessing we want to replace "../" and "..\", not only ".."?
Otherwise "../etc" turns to "/etc", which might be an actual path on the filesystem.

Just a guess though, I'm not sure where this is used from.

Reply:
@GermanBluefox ?!